CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [v2.0.1](https://github.com/liferay/alloy-editor/tree/v2.0.1) (2019-04-26)
+
+[Full changelog](https://github.com/liferay/alloy-editor/compare/v2.0.0...v2.0.1)
+
+- fix: Remove format only where styleFn does not exist ([\#1249](https://github.com/liferay/alloy-editor/pull/1249))
+- chore: Prepare for release v2.0.0 ([\#1247](https://github.com/liferay/alloy-editor/pull/1247))
```

Comment:
You could drop this line, I think, as it isn't really anything related to this release. (Separately, we might want to update the changelog generator to ignore merge commits that coincide exactly with the previous version).

In practice, I haven't actually been merging commits like this one: I just submit them as draft PRs so that I can see the PR pass, and then I just `git push upstream master --ff-only` them. ([Process documented for the liferay-js-themes-toolkit here](https://github.com/liferay/liferay-js-themes-toolkit/blob/a20ea987d20e20c828c6412dfa10ad27a292102a/CONTRIBUTING.md#4-send-a-release-pr), although I didn't describe it in the [alloy-editor "Contributing" wiki](https://github.com/liferay/alloy-editor/wiki/Contributing)).

Reply:
Ok thanks @wincent

Reply:
(Derp, no such thing as `--ff-only` on `git push`. Sorry.)